aspnetcore/publishing/iis.md

```diff
@@ -170,41 +174,64 @@ In web farm scenarios, an application can be configured to use a UNC path to sto
 
 In most cases, enabling application logging will assist in troubleshooting problems with application or the reverse proxy. See [Logging](../fundamentals/logging.md) for more information.
 
+Our last troubleshooting tip pertains to apps that fail to run after upgrading either the .NET Core SDK on the development machine or package versions within the app. In some cases, incoherent packages may break an app when performing major upgrades. You can fix most of these issues by deleting the `bin` and `obj` folders in the project, clearing package caches at `%UserProfile%\.nuget\packages\` and `%LocalAppData%\Nuget\v3-cache`, restoring the project, and confirming that your prior deployment on the server has been completely deleted prior to re-deploying the app.
+
+>[!TIP]
+> A convenient way to clear package caches is to obtain the `NuGet.exe` tool from [NuGet.org](https://www.nuget.org/), add it to your system PATH, and execute `nuget locals all -clear` from a command prompt.
+
 Common errors and general troubleshooting instructions:
 
 ### Installer unable to obtain VC++ Redistributable
 
-* **Installer Exception:** Installation of the .NET Core Windows Server Hosting Bundle fails with *0x80070002 - The system cannot find the file specified*.
+* **Installer Exception:** 0x80072efd or 0x80072f76 - Unspecified error
+
+* **Installer Log Exception\*:** Error 0x80072efd or 0x80072f76: Failed to execute EXE package
+
+\*The log is located at C:\Users\\{USER}\AppData\Local\Temp\dd_DotNetCoreWinSvrHosting__{timestamp}.log.
+
+Troubleshooting:
+
+* If the server does not have Internet access while installing the server hosting bundle, this exception will occur when the installer is prevented from obtaining the *Microsoft Visual C++ 2015 Redistributable (x64)*. You may obtain an installer from the [Microsoft Download Center](https://www.microsoft.com/en-us/download/details.aspx?id=53840). If the installer fails, you may not receive the .NET Core runtime required to host portable applications. If you plan to host portable applications, confirm that the runtime is installed in Programs &amp; Features. You may obtain a runtime installer from [.NET Downloads](https://www.microsoft.com/net/download/core). After installing the runtime, restart the server or restart IIS by executing **net stop was /y** followed by **net start w3svc** from a command prompt.
```

Comment:
I believe the current terminology for "portable applications" is "framework-dependent deployments" and "self-contained applications" are "self-contained deployments" -- https://docs.microsoft.com/en-us/dotnet/articles/core/deploying/